src/Providers/OpenRouter/Handlers/Structured.php

```diff
@@ -66,6 +66,7 @@ protected function sendRequest(Request $request): array
                         'schema' => $request->schema()->toArray(),
                     ],
                 ],
+                'provider' => $request->providerOptions('provider') ?? null,
```

Comment:
NIT: Is there a reason for defaulting to null rather than omitting the field if not set? (Some APIs prefer missing keys over explicit nulls.)